.github/workflows/cron-npm-install.yml

```diff
@@ -1,40 +0,0 @@
-name: NPM install testing workflow
-# By default the sha where it runs is latest commit on default branch
-# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
-on:
-  schedule:
-    # daily at 19:00 UTC
-    - cron: 0 19 * * *
-  workflow_dispatch:
-  pull_request:
-    branches:
-      - master
-
-jobs:
-  install-npm-package:
-    name: ${{ matrix.package }} NPM package install
-    runs-on: ubuntu-latest
-    container:
-      image: node:18-bullseye
-    strategy:
-      fail-fast: false
-      matrix:
-        package:
-          - '@celo/utils@beta'
-          - '@celo/contractkit@beta'
-          - '@celo/celocli@beta'
-    steps:
-      - name: Install @celo/celocli dependencies
-        if: matrix.package == '@celo/celocli@beta'
-        run: |
-          apt update
-          apt install -y libusb-1.0-0-dev libudev-dev
-          npm install node-gyp --global
-          git config --global url."https://".insteadOf ssh://
-
-      - name: Installing npm package
-        run: npm install ${{ matrix.package }} --global
-      - name: Test celocli command
-        if: matrix.package == '@celo/celocli@beta'
-        run: |
-          celocli account:new
```

Comment:
moved https://github.com/celo-org/developer-tooling/pull/438/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR71

so that it runs at a logical time